pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -61,6 +61,9 @@ type LoggingSpec struct {
 	GlobalFilters []Filter `json:"globalFilters,omitempty"`
 	// Limit namespaces to watch Flow and Output custom resources.
 	WatchNamespaces []string `json:"watchNamespaces,omitempty"`
+	// Cluster domain name to be used when templating URLs to services (default: "cluster.local").
+	// +kubebuilder:validation:Optional
+	ClusterDomain string `json:"clusterDomain"`
```

Comment:
I think we should go with an optional string here to allow explicitly setting it to empty. This way, the user can force resolution of the "clusterless" domain name.
```suggestion
	ClusterDomain *string `json:"clusterDomain,omitempty"`
```